sdk/cognitiveservices/azopenai/example_client_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azopenai_test
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/cognitiveservices/azopenai"
+)
+
+func ExampleNewClientForOpenAI() {
+	// NOTE: this constructor creates a client that connects to the public OpenAI endpoint.
```

Comment:
I think this comment should be moved to above the call to `azopenai.NewClientForOpenAI`

Reply:
Oops, yeah, that was weird.